app/javascript/shared/types/ContentBlock.re

```diff
@@ -9,11 +9,12 @@ type embedCode = option(string);
 type filename = string;
 type requestSource = [ | `User | `VimeoUpload];
 type lastResolvedAt = option(Js.Date.t);
+type width = [ | `auto | `lg | `md | `sm | `xl | `xl2 | `xs];
```

Comment:
I went with polymorphic variant because the latest bucklescript would convert it to strings in JavaScript .

Reply:
> I went with polymorphic variant because the latest bucklescript would convert it to strings in JavaScript .

Okay. However, the readability of the output JS isn't the most important thing for us.

Switching to regular variants would make the Reason code _simpler_ which, I think, is more important than the syntax of the output JS. 🤔 

The reason I'm saying we don't need polymorphic variants here is that we're not really _using_ the **polymorphic** feature of these _polymorphic_ variants anywhere... 😅 

Here's a good discussion of [when to use polymorphic variants](https://dev.realworldocaml.org/variants.html#scrollNav-4-3).